circle.yml

```diff
@@ -11,6 +11,10 @@ general:
     - http-servers/build/libs
     - ssl-config/build/libs
 
+dependencies:
+  override:
+    - ./gradlew resolveDependencies
```

Comment:
You'd have to catch up with Yun. Maybe he'd like some help? :)

Sent from my iPhone

On Apr 11, 2016, at 5:21 PM, Nick Miyake <notifications@github.com<mailto:notifications@github.com>> wrote:


In circle.yml<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_http-2Dremoting_pull_100-23discussion-5Fr59303828&d=CwMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=Z19FZ4sKAcw3cGtugYemPsnuYG-xLo9dLIoRe2x-fkk&s=MCRgbCcXhlB2QRbuqA7i6HZHpgE8dYg4IRBIpQz0k24&e=>:

> @@ -11,6 +11,10 @@ general:
>      - http-servers/build/libs
>      - ssl-config/build/libs
>
> +dependencies:
> +  override:
> +    - ./gradlew resolveDependencies


Is there any code that exists for that, or is it something that's been planned/thought of but not yet written?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_http-2Dremoting_pull_100_files_5224a2d46b443c85fcb5e852c1d2ed7be5411d34-23r59303828&d=CwMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=Z19FZ4sKAcw3cGtugYemPsnuYG-xLo9dLIoRe2x-fkk&s=xeIIGfHw-UsQ7vTIBjiRb0RYERZBSkniIsxDIdZiby0&e=>